api/dependencies.py

```
@@ -0,0 +1,17 @@
+from typing import Tuple
+
+from fastapi import File, Form, UploadFile
+from schemas import DashboardItemCreate
+
+
+def parse_dashboard_form(
+    title: str = Form(...),
+    description: str | None = Form(None),
+    image: UploadFile | str | None = File(None),
```

Ruff (0.12.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)